### DISCUSSION TITLE
Quick filter for Projects list

### DISCUSSION THREAD

**Discussion by mr-robot-o** [nrwl/nx-console#1861] [Feature Requests]
It would be great if the projects list could have a quick filter since NX workspaces can start to accumulate a large number of projects quickly once you begin building shared libs.  Ideally a quick filter search input would be nice to filter the list down quickly based on a string you would like to see all the projects related to.

<img width="349" alt="projects-searchable" src="https://github.com/nrwl/nx-console/assets/266783/3497f169-08a3-455d-99cf-a8000229be75">

**Comment by simonbergstrom** [comment]
+ 1 on this feature request. Would really easy our day to day work in our monorepo with +20 apps and +100 libs. Gets really hard to find in the long list so an easy filter and/or a search field would be fanstastic

**Comment by MaxKless** [comment]
Hey! Sorry saw this super late. 
You should be able to do exactly this by typing Ctrl + F. 
Does that work for you?
